python/src/sdf/pyWorld.cc

```diff
@@ -0,0 +1,173 @@
+  //  - Added Light methods
+  //  - Added Physics methods
+
+  pybind11::class_<sdf::World> geometryModule(module, "World");
+  geometryModule
+    .def(pybind11::init<>())
+    .def(pybind11::init<sdf::World>())
+    .def("validate_graphs", &sdf::World::ValidateGraphs,
+         "Check that the FrameAttachedToGraph and PoseRelativeToGraph "
+         "are valid.")
+    .def("name", &sdf::World::Name,
+         "Get the name of model.")
+    .def("set_name", &sdf::World::SetName,
+         "Set the name of model.")
+    .def("audio_device", &sdf::World::AudioDevice,
+         "Get the audio device name. The audio device can be used to "
+         "playback audio files. A value of \"default\" or an empty string "
+         "indicates that the system's default audio device should be used.")
+    .def("set_audio_device", &sdf::World::SetAudioDevice,
+         "Set the audio device name. See std::string AudioDevice() const "
+         "for more information.")
+    .def("wind_linear_velocity", &sdf::World::WindLinearVelocity,
+         "Get the wind linear velocity in the global/world coordinate "
+         "frame. Units are meters per second \f$(\frac{m}{s})\f$")
+    .def("set_wind_linear_velocity", &sdf::World::SetWindLinearVelocity,
+         "Set the wind linear velocity in the global/world coordinate "
+         "frame. Units are meters per second \f$(\frac{m}{s})\f$")
+    .def("gravity", &sdf::World::Gravity,
+         "Get the acceleration due to gravity. The default value is "
+         "Earth's standard gravity at sea level, which equals "
+         "[0, 0, -9.80665] \f$(\frac{m}{s^2})\f$")
+    .def("set_gravity", &sdf::World::SetGravity,
+         "Set the acceleration due to gravity. Units are meters per "
+         "second squared \f$(\frac{m}{s^2})\f$")
+    .def("magnetic_field", &sdf::World::MagneticField,
+         "Get the magnetic vector in Tesla, expressed in "
+         "a coordinate frame defined by the SphericalCoordinates property. "
+         "A spherical coordinate can be specified in SDF using the "
+         "<spherical_coordinates> element.")
+    .def("set_magnetic_field", &sdf::World::SetMagneticField,
+         "Set the magnetic vector in Tesla, expressed in "
+         "a coordinate frame defined by the SphericalCoordinate. "
+         "A spherical coordinate can be specified in SDF using the "
+         "<spherical_coordinates> element.")
+    .def("spherical_coordinates", &sdf::World::SphericalCoordinates,
+         pybind11::return_value_policy::reference,
+         "Get the spherical coordinates for the world origin.")
+    .def("set_spherical_coordinates", &sdf::World::SetSphericalCoordinates,
+         "Set the spherical coordinates for the world origin.")
+    .def("model_count", &sdf::World::ModelCount,
+         "Get the number of explicit model that are immediate (not nested) "
+         "children of this World object.")
+    .def("model_by_index",
+         pybind11::overload_cast<uint64_t>(
```

Comment:
Which overload is this going to pick? Having the `pybind11::const_` tag makes 
me think it will pick the const function, but I'm not sure. In the other 
bindins, I think we've been picking the non-const overload.